### PR TITLE
Adding release notes

### PR DESCRIPTION
**Pull request by erundle**
Adding information to readme on how to cut a release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,35 @@ grunt ngdocs:view
 
 After executing this tasks you'll be able to access the documentation at [http://localhost:8000/](http://localhost:8000/).
 
+## Releasing
+
+Angular PatternFly is released through the Bower
```
Comment (waldenraines) [inline README.md]: I think it's just "Bower" :smile:

File: README.md [hunk 2]
```diff
@@ -54,6 +54,35 @@ grunt ngdocs:view
...
+Update the version listed in `package.json` by editing the file and changing the line:
+
+```
+"version": "<new_version>"
+```
+
+Commit the version bump:
+
+```
+git commit -m"Version bump to <new_version>"
```
Comment (waldenraines) [inline README.md]: Missing a space between the `-m` and the message.  This still works but I haven't seen command line arguments written this way before.

File: README.md [hunk 3]
```diff
@@ -54,6 +54,35 @@ grunt ngdocs:view
...
+```
+
+Publish a new set of release notes with the "new version" as the tag:
```
Comment (waldenraines) [inline README.md]: Maybe "Publish a new set of release notes with `new_version` as the tag version:`?

**Comment by waldenraines** [comment]
A couple of nitpicks but otherwise LGTM.

**Comment by erundle** [comment]
Fixed all the changes and squashed including "The Bower" :)

**Pull request merged**